.reuse/dep5

```diff
@@ -0,0 +1,17 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: A3 Audio
+Upstream-Contact: A3 Audio <contact@a3-audio.com>
+Source: https://github.com/a3-audio/a3-core
+Comment: https://a3-audio.com
+
+Files: docs/*
+Copyright: 2023 A3 Audio <contact@a3-audio.com>
+License: CC-BY-SA-4.0
+
+Files: platform-config/*
+Copyright: 2023 A3 Audio <contact@a3-audio.com>
+License: CC-BY-SA-4.0
+
+Files: engine/*
+Copyright: 2023 A3 Audio <contact@a3-audio.com>
+License: GPL-3.0-or-later
```

Comment:
In this repo, the engine basically consists ouf of the Reaper project. I think it's not meaningful to license that under GPL since it is not program code. I would suggest CC-BY-SA.

Reply:
OK that was nonsense. There are of course also the scripts written in Python and the Supercollider stuff. So my suggestion would be GPL for those and CC-BY-SA for the reaper project.